infrastructure/docker/traffic_portal/Dockerfile

```diff
@@ -22,7 +22,7 @@
 
 FROM centos/systemd
 
-RUN curl -sL https://rpm.nodesource.com/setup_6.x | bash -
+RUN curl -sL https://rpm.nodesource.com/setup_12.x | bash -
```

Comment:
Shouldn't `infrastructure/cdn-in-a-box/traffic_portal/Dockerfile` also have this change?